keadm/README.md

```diff
@@ -1,17 +1,258 @@
+  -s, --server string                              IP:Port address of cloud components host/VM
+
+```
+
+1. For Docker, K8S and KubeEdge flags the functionality is same as mentioned in `kubeedge init`
+2. -s, --server, It should be in the format <IPAddress:Port>, where the default port is 8080. Please see the example above.
+                It is a mandatory flag
+
+**IMPORTANT NOTE:** The versions used for Docker, KubeEdge and K8S, should be same in both Cloud and Edge side.
+
+## Reset KubeEdge Cloud and Edge components
+
+Referring to `KubeEdge Installer Doc`, the command to stop KubeEdge cloud (edge controller). It doesn't uninstall/remove any of the pre-requisites.
+
+- Execute `sudo kubeedge reset`
+
+### Command flags
+
+```
+sudo kubeedge reset --help
+
+kubeedge reset command can be executed in both cloud and edge node
+In cloud node it shuts down the cloud processes of KubeEdge
+In edge node it shuts down the edge processes of KubeEdge
+
+Usage:
+  kubedge reset [flags]
+
+Examples:
+
+For cloud node:
+kubeedge reset
+
+For edge node:
+kubeedge reset --server 10.20.30.40:8080
+    - For this command --server option is a Mandatory option
+
+
+Flags:
+  -h, --help            help for reset
+  -s, --server string   IP:Port address of cloud components host/VM
+
+```
+
+## Simple steps to bring up a KubeEdge setup and deploy a pod
+**NOTE:** All the below steps are executed as root user, to execute as sudo user ,Please add **sudo** infront of all the commands
+### 1.EdgeController (With K8S API-Server)
+##### Install tools with the particular version
+```
+kubeedge init --kubeedge-version=<kubeedge Version>  --kubernetes-version=<kubernetes Version> --docker-version=<Docker version>
+```
+##### Install tools with the Default version version
+```
+kubeedge init --kubeedge-version= --kubernetes-version= --docker-version
+or
+kubeedge init --kubeedge-version --kubernetes-version --docker-version
+```
+
+**NOTE:**
+kubeadm join **192.168.20.134**:6443 --token 2lze16.l06eeqzgdz8sfcvh \
+         --discovery-token-ca-cert-hash sha256:1e5c808e1022937474ba264bb54fea42b05eddb9fde2d35c9cad5b83cf5ef9ac  
+         After Kubeedge init ,please note the **CloudIp** as highlighted above generated from console output and port is **8080**.
+
+### 2.Manually copy Certs.tgz. to /etc/kubeedge in Edge vm
+
+In Edge VM
+```
+mkdir -p /etc/kubeedge
+```
+In Cloud VM
+
+```
+scp -r certs.tgz username@ipEdgevm:/etc/kubeedge
+```
+
+In Edge VM untar the certs,tgz file
+
+```
+cd /etc/kubeedge
+tar -xvzf certs.tgz
+```
+
+### 3.Edge Node join
+##### Install tools with the Default version 
+```
+kubeedge join --kubeedge-version=<kubeedge Version>  --kubernetes-version=<kubernetes Version> --docker-version=<Docker version> --server=<CloudIp:8080>
+```
+
+```
+kubeedge join --kubeedge-version= --kubernetes-version= --docker-version= --server=CloudIp:8080
```

Comment:
I dont think this is very friendly. If a flag is not mentioned, defaults should be taken

Reply:
yes if you dont mention the flags default version only will be taken..Just i mentioned in Readme so that user could be familiar about availabe flags !!.

Reply:
yes if you dont mention the flags default version only will be taken..Just i mentioned in Readme so that user could be familiar about availabe flags !!.
![join](https://user-images.githubusercontent.com/39593583/57597773-54bc7500-756e-11e9-909b-ecfa25866a3d.png)
![kubeedgeinit](https://user-images.githubusercontent.com/39593583/57597776-571ecf00-756e-11e9-921e-995656bb06aa.png)

if you see above in the image attached..Both init and join works and default tools(kubernetes,docker) will be used if we didnt specify the flag

Reply:
We have already mentioned supported flags. I think we can remove this line

Reply:
ok